corporate/lib/registration.py

```diff
@@ -105,18 +108,12 @@ def check_spare_licenses_available_for_registering_new_user(
 def check_spare_licenses_available_for_inviting_new_users(
     realm: Realm, extra_non_guests_count: int = 0, extra_guests_count: int = 0
 ) -> None:
-    num_invites = extra_non_guests_count + extra_guests_count
     plan = get_plan_if_manual_license_management_enforced(realm)
     if plan is None:
         return
 
     try:
         check_spare_licenses_available(realm, plan, extra_non_guests_count, extra_guests_count)
     except LicenseLimitError:
-        if num_invites == 1:
-            message = _("All Zulip licenses for this organization are currently in use.")
-        else:
-            message = _(
-                "Your organization does not have enough unused Zulip licenses to invite {num_invites} users."
```

Comment:
These messages aren't quite accurate if the failure is due to renewal licenses being set to the number of licenses currently in use. Instead of trying to tailor the message for each case, I went with one simplified message to use for all cases:

"Your organization does not have enough Zulip licenses to invite new users."